app/frontend/wallet/constants.ts

```diff
@@ -116,6 +116,4 @@ export const TREZOR_ERRORS = {
   [CryptoProviderFeature.POOL_OWNER]: InternalErrorReason.TrezorPoolRegNotSupported,
 }
 
-export const MAX_ACCOUNT_INDEX = 30
-
-export const MAX_BULK_EXPORT_AMOUNT = 5
+export const MAX_ACCOUNT_INDEX = 34
```

Comment:
why did we have to bump this? I'd rather stay at 30 if we don't have a strong reason to bump it, 34 seems really arbitrary